discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
```

Comment:
In what way the lib's classes and methods are documented. I wasn't sure what name to use so I chose this. This is supposed to be for docs that list every method with parameters and what it does. Possible values include `inline` for inline docs using JavaDoc, YARD etc., `external` for manually updated docs with equivalent content and `none` for libs that don't use this kind of documentation.